_config.yml

```diff
@@ -0,0 +1,37 @@
+# Welcome to Jekyll!
+#
+# This config file is meant for settings that affect your whole blog, values
+# which you are expected to set up once and rarely edit after that. If you find
+# yourself editing this file very often, consider using Jekyll's data files
+# feature for the data you need to update frequently.
+#
+# For technical reasons, this file is *NOT* reloaded automatically when you use
+# 'bundle exec jekyll serve'. If you change this file, please restart the server process.
+
+# Site settings
+# These are used to personalize your new site. If you look in the HTML files,
+# you will see them accessed via {{ site.title }}, {{ site.email }}, and so on.
+# You can create any custom variable you would like, and they will be accessible
+# in the templates via {{ site.myvariable }}.
+
+url: "http://hierplane.allenai.org" # the base hostname & protocol for your site, e.g. http://example.com
```

Comment:
Let's set this to `allenai.github.io/hierplane` for now.  We can switch this once the new DNS is in place.  I'd rather get the site up first, then migrate DNS once we know it's working.

Reply:
Will do.

Reply:
Done in latest commit.